src/app/qgisapp.h

```diff
@@ -1887,6 +1887,13 @@ class APP_EXPORT QgisApp : public QMainWindow, private Ui::MainWindow
                                            const QString &providerKey, bool guiWarning,
                                            bool guiUpdate );
 
+    /**
+     * This method will open a dialog so the user can select MDAL mesh sublayers to load,
+     * and then returns a list of these layers. Layers are added to the project in the method.
+     * \since QGIS 3.14
```

Comment:
this is APP_EXPORT, comments are optional and you do not need to add \since.

Reply:
except for the "since", consistent with other surrounding methods

Reply:
Good.